### PR TITLE
Update dependencies and apply new eslint rules

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

File: src/helperFunctions.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ class helpers {
    * @returns {Promise.<error>}           string: error message, if any.
    */
   _updateCurrentUser () {
-    let self = this
+    const self = this
```
Comment (kulmann) [inline src/helperFunctions.js:116]: I know that you didn't touch it, but `self` should not be necessary here, as `this` is automatically bound as context in lambda functions.

File: package.json [hunk 2]
```diff
@@ -122,7 +122,7 @@
   },
   "lint-staged": {
     "*.js": [
-      "lint --fix",
+      "./node_modules/.bin/eslint --fix",
```
Comment (kulmann) [inline package.json:125]: why not use the `lint-fix` script?

Comment (DeepDiver1975) [reply package.json:125]: lint --fix as well as lint-fix are failing without output - no idea why.
this way it is working ... :man_shrugging:

File: src/helperFunctions.js (hunk 1 above)
Comment (DeepDiver1975) [reply src/helperFunctions.js:116]: Can we agree to change this in a follow up PR?

Comment (kulmann) [reply src/helperFunctions.js:116]: 👍

**Pull request merged**